cni/ipam/ipam.go

```diff
@@ -138,8 +138,8 @@ func (plugin *ipamPlugin) Add(args *cniSkel.CmdArgs) error {
 	var result *cniTypesCurr.Result
```

Comment:
As discussed please update readme files + also include scripts  (with crictl) commands that used to test the changes/
Update the PR Title - include process based containers.

Reply:
Updated the PR title. I have a separate PR for updating the document ( since it's convenient to update the document via github UI and it didn't give me an option to create an update in this PR. ) Here is the PR that will have the documentation update: https://github.com/Azure/azure-container-networking/pull/423.
I would like to merge this PR once the test suit passes so that the host container can be moved against master.